app/presenters/document/field_group/section/value_presenter.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+module Document
+  module FieldGroup
+    module Section
+      ##
+      # Presenter for one value of a metadata field
+      #
+      # TODO: should this be sub-classed? e.g. for single string values, language
+      #   maps, entity-linked values
+      class ValuePresenter < ApplicationPresenter
+        include BlacklightDocumentPresenter
+        include EntitiesHelper
+        include UrlHelper
+
+        attr_reader :document, :controller, :section, :content, :field, :entity
+        delegate :blank?, :nil?, :present?, :empty?, to: :text
+
+        # @param document [Europeana::Blacklight::Document]
+        # @param controller [ActionController::Base]
+        # @param section [Document::FieldGroup::SectionPresenter]
+        # @param field [String,Symbol]
+        # @param content [String]
+        # @param entity [Europeana::Blacklight::Document]
+        def initialize(document, controller, section, field, content, entity = nil)
+          @document = document
+          @controller = controller
+          @section = section
+          @field = field
+          @content = content
+          @entity = entity
+        end
+
+        def display
+          {
+            text: text,
+            url: url,
+            ga_data: section.ga_data,
+            extra_info: extra_info
+          }
+        end
+
+        def extra_info
+          return nil unless for_entity? && section.entity_extra.present?
+          return nil unless entity.present?
+          nested_hash(section.entity_extra, entity)
+        end
+
+        def for_entity?
+          entity.present?
+        end
+
+        def url
+          if linkable_for_entity?
+            url = entity_url
+            return url unless url.nil?
+          end
+
+          return search_path if section.search_field.present?
+
+          linkable_value? ? content : nil
+        end
+
+        def linkable_value?
+          return false unless content.is_a?(String)
+          content.start_with?('http://', 'https://')
+        end
+
+        def linkable_for_entity?
+          Rails.application.config.x.enable.entity_page &&
```

Comment:
should we remove this configuration already?

Reply:
Yes, and done in bed709e0